src/libexpr/eval.cc

```diff
@@ -161,12 +162,12 @@ const Value *getPrimOp(const Value &v) {
 }
 
 
-string showType(const Value & v)
+string showType(ValueType type)
 {
-    switch (v.type) {
+    switch (type) {
         case tInt: return "an integer";
-        case tBool: return "a boolean";
-        case tString: return v.string.context ? "a string with context" : "a string";
+        case tBool: return "a Boolean";
```

Comment:
pardon me if I don't get it but why does the `b` have to be capitalized in this case?

Reply:
Because "Boolean" comes from a name (George Boole).